config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+  get "/studios/:id", to: "studios#show"
+  get "/movies/:id", to: "movies#show"
+
```

Comment:
Using what I stated above about the `:post` method, as well as the extra controller, our route could then look like

```ruby
post "/movie_actors", to: "movie_actors#create"
```
The path could always change to `/movies/:id/actors` too, there are lots of options here. In any case, we should see a `post` method when creating.